pkg/services/ngalert/metrics/remote_alertmanager.go

```diff
@@ -0,0 +1,37 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/weaveworks/common/instrument"
+)
+
+type RemoteAlertmanager struct {
+	HTTPRequestsTotal    *prometheus.CounterVec
+	HTTPRequestsFailed   *prometheus.CounterVec
+	HTTPRequestsDuration *instrument.HistogramCollector
+}
+
+func NewRemoteAlertmanagerMetrics(r prometheus.Registerer, subsystem string) *RemoteAlertmanager {
+	return &RemoteAlertmanager{
+		HTTPRequestsTotal: promauto.With(r).NewCounterVec(prometheus.CounterOpts{
+			Namespace: Namespace,
+			Subsystem: subsystem,
+			Name:      "remote_alertmanager_http_requests_total",
+			Help:      "Number of HTTP requests sent to the remote Alertmanager.",
+		}, []string{"method", "path", "status_code"}),
+		HTTPRequestsFailed: promauto.With(r).NewCounterVec(prometheus.CounterOpts{
+			Namespace: Namespace,
+			Subsystem: subsystem,
+			Name:      "remote_alertmanager_http_requests_failed",
+			Help:      "Number of HTTP requests sent to the remote Alertmanager that failed.",
+		}, []string{"method", "path"}),
+		HTTPRequestsDuration: instrument.NewHistogramCollector(promauto.With(r).NewHistogramVec(prometheus.HistogramOpts{
+			Namespace: Namespace,
+			Subsystem: subsystem,
+			Name:      "remote_alertmanager_http_request_duration_seconds",
```

Comment:
Not sure whether to use `request` or `requests`, decided on the singular form after taking a look at what we do in other places (e.g. [historian](https://github.com/grafana/grafana/blob/feb7b38fba1f8d694395efa803c742d2678eab7e/pkg/services/ngalert/metrics/historian.go#L13-L15)).